### PR TITLE
Mod5

### PR DESCRIPTION
**Pull request by kcny**

### CODE REVIEW

File: app/views/admin/videos/new.html.haml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+%section.admin_add_video
+  .container
+    .row
+      .col-md-10.col-md-offset-1
+        = bootstrap_form_for [:admin, @video], html: { class: "form-horizontal" } do |f|
```
Comment (brandoncc) [inline app/views/admin/videos/new.html.haml]: This is not the right way to do horizontal forms with bootstrap_form_for anymore. The syntax is now `layout: :horizontal`. You can see it in the gem docs.

File: app/views/admin/videos/new.html.haml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      .col-md-10.col-md-offset-1
+        = bootstrap_form_for [:admin, @video], html: { class: "form-horizontal" } do |f| 
+          %ul.nav.nav-tabs
+            %li
+              %a(href="/ui/admin_views_payments") Recent Payments
+            %li.active
+              = link_to "Add New Video", new_admin_video_path
+          %br
+          %fieldset
+            = f.text_field :title, class: "span3"
```
Comment (brandoncc) [inline app/views/admin/videos/new.html.haml]: Once you fix the issue above you won't need to specify these classes anymore. By the way `span3 is not a valid class in Bootstrap 3.

File: db/seeds.rb [hunk 3]
```diff
@@ -37,111 +37,111 @@
 Category.create!(name: "Adult Animation")
 Category.create!(name: "Comedy")
 
-#Videos
+# #Videos
 
-      # Comedy
+#       # Comedy
```
Comment (brandoncc) [inline db/seeds.rb]: Why is the rest of this file commented out?

File: spec/controllers/admin/videos_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    it_behaves_like "requires admin" do 
+      let(:action) { get :new }
+    end 
+    it "sets the @video to a new video" do 
+      set_current_admin
+      get :new
+      expect(assigns(:video)).to be_instance_of Video
+      expect(assigns(:video)).to be_new_record 
+    end
+    # it "redirects the regular user the the home path" do
```
Comment (brandoncc) [inline spec/controllers/admin/videos_controller_spec.rb]: Make sure you remove commented/unused code form from your codebase.

File: spec/controllers/admin/videos_controller_spec.rb [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+    it_behaves_like "requires login" do
+      let(:action) { get :new }
+    end
+    it_behaves_like "requires admin" do 
+      let(:action) { get :new }
+    end 
+    it "sets the @video to a new video" do 
+      set_current_admin
+      get :new
+      expect(assigns(:video)).to be_instance_of Video
```
Comment (brandoncc) [inline spec/controllers/admin/videos_controller_spec.rb]: You could also say `be_a_new(Video)`.

File: app/views/admin/videos/new.html.haml (hunk 1 above)
Comment (kcny) [inline app/views/admin/videos/new.html.haml]: Okay, sorted, thanks!

File: app/views/admin/videos/new.html.haml (hunk 2 above)
Comment (kcny) [inline app/views/admin/videos/new.html.haml]: Done!

File: db/seeds.rb (hunk 3 above)
Comment (kcny) [inline db/seeds.rb]: I wanted to leave out the videos from the data base just one time.  I've removed the comments.

File: spec/controllers/admin/videos_controller_spec.rb (hunk 4 above)
Comment (kcny) [inline spec/controllers/admin/videos_controller_spec.rb]: Done.

File: spec/controllers/admin/videos_controller_spec.rb (hunk 5 above)
Comment (kcny) [inline spec/controllers/admin/videos_controller_spec.rb]: Updated.  Thanks, Brandon!

**Pull request merged**